metroae

```diff
@@ -86,6 +86,7 @@ fi
 if [[ -z $SETUP_FILE ]]; then
     SETUP_FILE=/opt/metroae/.metroae
 fi
+
```

Comment:
Can you create a pull for dev branch instead of Master. That will take out this file